packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -358,7 +358,7 @@ const main = async () => {
                 warnings.forEach(warn)
                 if (wait) {
                     success('Bundle Uploaded - waiting for deployment to complete')
-                    await client.waitForDeploy()
+                    await client.waitForDeploy(projectSlug, target)
```

Comment:
This fixes the `--wait` command even outside of Github actions.

Without this, using `--wait` will result in an HTTP 403 as we poll `api/projects/undefined/target/undefined/`.